models/DataObject/AbstractObject.php

```diff
@@ -966,6 +983,39 @@ public function getChildrenSortOrder(): string
         return $this->childrenSortOrder ?? self::OBJECT_CHILDREN_SORT_ORDER_DEFAULT;
     }
 
+    /**
+     * @return $this
+     */
+    public function setClassId(string $classId): static
+    {
+        $this->classId = $classId;
+
+        return $this;
+    }
+
+    public function getClassId(): ?string
+    {
+        if (isset($this->classId)) {
```

Comment:
@cuca24 thank you for moving the method and the property. I am thinking about also improving the method itself.
Since the setClassId has now a string parameter, we could remove the string cast from the getter. Also we do not need the `isset` anymore right? Since classId always exists and it returns null anyways if the setter is not called.
WDYT?

Reply:
@mattamon You are totally right. PR updated :)